java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -379,6 +379,29 @@ public byte getByte(long index) {
     return offHeap.data.getByte(index * type.sizeInBytes);
   }
 
+  /**
+   * Get the byte list at index.
+   */
+  public byte[] getByteList(long index) {
```

Comment:
I think we need to document this a little bit better because how nulls are handled in a child is not clear from the javadocs or from the signature of the function.